src/madetomeasure_api/handler.clj

```diff
@@ -53,7 +53,8 @@
   (-> (routes
         home-routes
         campaign-routes
-        subscriber-routes
+        validated-subscriber-routes
```

Comment:
I futzed around with this a bit. There's some things that need to be validated and 400ed before we do anything. Some that we don't have to.